### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adrienzieba**

### CODE REVIEW

**Comment by adrienzieba** [comment]
test comment

**Comment by adrienzieba** [comment]
fsdf

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# status_check_test
\ No newline at end of file
+# status_check_test
+
```
Comment (adrienzieba) [inline README.md:2]: fsdfg

**Comment by adrienzieba** [comment]
fdsaf

**Comment by adrienzieba** [comment]
jkhgj

**Comment by adrienzieba** [comment]
tete

**Comment by adrienzieba** [comment]
anothertest

**Comment by adrienzieba** [comment]
fsadgf

**Comment by adrienzieba** [comment]
new test d

**Comment by adrienzieba** [comment]
edfaswdefgasdfLGTM robotegwqergwerhgwerh

**Comment by adrienzieba** [comment]
dd

**Comment by adrienzieba** [comment]
tu

**Comment by adrienzieba** [comment]
fafe

**Comment by ghost** [comment]
LGTM robot

**Comment by adrienzieba** [comment]
dwda

**Comment by adrienzieba** [comment]
sdfsadgf

**Comment by adrienzieba** [comment]
gdgd

**Comment by adrienzieba** [comment]
vvv

**Comment by patvong** [comment]
LGTM robot

**Comment by patvong** [comment]
sdfsf

**Comment by adrienzieba** [comment]
affeef

File: README.md (hunk 1 above)
Comment (adrienzieba) [inline README.md:2]: additional info

Comment (adrienzieba) [inline README.md:2]: :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# status_check_test
```
Comment (adrienzieba) [inline README.md:1]: why that

**Pull request closed without merging**